jenkins_jobs/trigger_host_os_build_from_builds_repo.groovy

```diff
@@ -23,6 +23,7 @@ job('trigger_host_os_build_from_builds_repo') {
       extensions {
 	commitStatus {
 	  context('Build Host OS')
+	  statusUrl('${JENKINS_URL}/job/build_host_os/${TRIGGERED_BUILD_NUMBER_build_host_os}')
```

Comment:
I was about to do this but then I realized all the variables from this file come from Jenkins so I don't see why give this one special treatment.